### ISSUE TITLE
Is there some difference in the baseline between Yours and PromptDet

### ISSUE THREAD

**Issue by xushilin1** [hanoonaR/object-centric-ovd#14]
The baseline's $AP_r$ of your and PromptDet are 12.2 and 7.4, respectively. But both of you seem to replace the original classification with softmax cross-entropy of Mask RCNN with similarity score with sigmoid cross entropy. Why is your baseline much higher PromptDet? 

Could you please give more details about the difference between your baseline and PromptDet's baseline?

Thanks for your reply.

**Comment by hanoonaR** [comment]
Hi @xushilin1 ,

For LVIS experiments, we use Mask R-CNN with ResNet-50FPN backbone with federated loss and sigmoid cross-entropy. We notice that the federated loss helps in improving the rare AP. Hope this helps.

Thank you.
